ydb/docs/en/core/public-talks.md

```diff
@@ -1,12 +1,21 @@
 # Public talks
 
+## 2023: Highload++ Serbia {#highload-serbia-2023}
+
+* Subject: Scale it easy: YDB's high performance in a nutshell.
+
+  * [About the event](https://highload.rs/2023/abstracts/9754).
+  * [Presentation](https://www.dropbox.com/scl/fi/p7yeo3ic06vcxjmfa7mua/4.Scale-it-easy-YDB-s-high-performance-in-a-nutshell_Evgeniy-Ivanov-Yandex-_ver.4.pdf?rlkey=r5dmdh614jz48wmdg8jbs6uw2&dl=0).
```

Comment:
@eivanov89 could you add this file to https://github.com/ydb-platform/ydb-presentations and use a link from there instead of someone's Dropbox?